spec/features/admin/repositories_spec.rb

```diff
@@ -68,7 +68,7 @@
       end
 
       it 'must have the form working' do
-        find('#repository_company_id').find(:option, repository.company).select_option
+        find('#repository_company_id').find(:option, repository.company.to_s).select_option
```

Comment:
I believe we should use `company.name` here to avoid indirection, I mean, in order to understand what exactly we're searching for we'll need to take a look at `Company#to_s `implementation. My point is, `to_s` is just a bad alias to company's name.